src/theming/config.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeThemeEventDetail, CHANGE_THEME_EVENT } from './theming-event';
+import { Theme } from './types';
+import { getAllCSSVariables } from './utils';
+
+let theme: Theme;
+
+type ThemeConfig = {
+  theme: Theme;
+  variant?: 'light' | 'dark';
```

Comment:
Do we need `variant`? It seems that we are not using it anywhere.

Reply:
AFAIK, we talked with @simeonoff that if we have to extend the API in the future and to pass the variant as well as the theme, it is better to have a single argument with optional props than to change the signature of the `configureTheme` function.

Reply:
We could add `variant` as optional parameter to the function which won't be a breaking change. I don't think we will have any other options for this function.

Reply:
True, I guess it's just a matter of _named_(config with typed signature) arguments vs _positional_(function args) ones. Whatever you say, I can go back to simply passing a string literal to the `configureTheme` function.

Reply:
Yep, it depends on the personal preferences. For up to 2-3 parameters I prefer the `func(arg1, arg2)` approach. 

Since we are going to release with a single parameter I would prefer `configureTheme('material')` than `configureTheme({ theme: 'material' })`.

Reply:
Also if we decide that we need a more complex config in the future we can change the signature like this:

```ts
configureTheme(theme: string | ThemeConfig)
```

Reply:
> Also if we decide that we need a more complex config in the future we can change the signature like this:
> 
> ```ts
> configureTheme(theme: string | ThemeConfig)
> ```

I still think is vastly better to be 
```typescript 
configureTheme(options: ThemeConfig)
```
and add new props to the options object than to union the type at a later stage. 
But as you pointed out it is a matter of personal preference.

Reply:
@rkaraivanov and @DiyanDimitrov, reverted it to the original implementation. The `configureTheme` expects an argument of type Theme now.  Obviously, if in JavaScript this will be a simple string.